### PR TITLE
Rename buildbot tool to buildcaptain 👩‍✈️

### PR DESCRIPTION
**Pull request by bobcatfish**
# Changes

Following up on #667 - we renamed build cop to build captain and have
been slowly updating all the references. This commit changes the name of
the tool itself, including the bucket it is built to and the deployment
configuration. In order for this to work, the deployment and the config
map will need to be updated manually as well.

I want to apply this manually and make sure it works (I think im gonna run into some kinda chicken and egg thing with changing the path to the csv - who set this up anyway - oh yeah it was me 🤣 )

/hold

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/ci/templates/plumbing-template.yaml [hunk 1]
```diff
@@ -416,7 +416,7 @@ spec:
         - name: gitHubCommand
           value: $(tt.params.gitHubCommand)
         - name: folders
-          value: bots/buildbot/config bots/mariobot/config pipelinerun-logs/config boskos gubernator prow robocat tekton
+          value: bots/buildconfig/config bots/mariobot/config pipelinerun-logs/config boskos gubernator prow robocat tekton
```
Comment (sbwsg) [inline tekton/ci/templates/plumbing-template.yaml:419]: should this be `buildconfig` or `buildcaptain`?

Comment (bobcatfish) [reply tekton/ci/templates/plumbing-template.yaml:419]: whoa thanks for catching that!

**Review by sbwsg — APPROVED** [review]

**Comment by bobcatfish** [comment]
Manually:

- Applied tekton/ci/templates/plumbing-template.yaml to update linting to look at the files in the new location (would break every other PR 😨 )
- Copied the existing buildcop-cfg configmap to buildcaptain-cfg
- Deleted buildcop-deployment
- [Deployed](https://github.com/tektoncd/plumbing/tree/fb3bd1adcc2ae1abde0826cab5ef65f7045c743c/bots/buildcaptain#deploying) with ` KO_DOCKER_REPO=gcr.io/tekton-releases/buildcaptain ko --context dogfood apply -f config/deployment.yaml`

I think it's ready to go!

image

It can't find the csv file b/c the path is hard coded (who did that? certainly not me... 😇 ) so we need to merge this PR before it can find it 🤞 

/hold cancel

**Pull request merged**